src/viewport/index.tsx

```diff
@@ -222,8 +224,12 @@ class Viewport extends React.Component<
             simulariumController,
             loadInitialData,
             onError,
+            stationaryViewport,
         } = this.props;
-        this.visGeometry.reparent(this.vdomRef.current);
+        this.visGeometry.setCanvasOnTheDom(
+            this.vdomRef.current,
+            stationaryViewport
+        );
```

Comment:
Very good change, was just looking at `reparent` when debugging mp4 recorder and this would have saved me time. Makes a search for 'canvas' in the repo more productive.